includes/adapter/OrderLineFromWooCommerce.php

```diff
@@ -29,8 +29,9 @@ public function __construct(WC_Order_Item $wcOrderItem)
     protected function prepareItemData(WC_Order_Item $wcOrderItem): array
     {
         $wcItemData = $wcOrderItem->get_data();
+        $wcProduct  = $wcOrderItem->get_product();
 
-        $price = (int) ($wcItemData['subtotal'] * 100.0);
+        $price = (int) ($wcProduct->get_price() * 100.0);
         $vat   = (int) ($wcItemData['subtotal_tax'] * 100.0);
```

Comment:
Let op dat in de orderregel nu de prijs per product staat (zoals bepaald door pps), maar de BELASTING nog steeds voor de hele regel!